pytab/core.py

```diff
@@ -300,6 +307,32 @@ def cli(
     system_info = hwi.get_system_info()
     click.echo(" success!")
 
+    # Logic for GPU Selection
+    gpus = system_info["gpu"]["gpu_elements"]
+    if len(gpus) > 1 and "gpu_id" not in globals():
+        click.echo("| Multiple GPU's detected")
+        for gpu in gpus:
+            click.echo(f"| {gpu["id"]}: {gpu["name"]} ")
+        gpu_id = click.prompt(
+            f"| please select a gpu id\n| Must be less than {len(gpus)}", type=int
+        )
+        if gpu_id > len(gpus):
+            click.echo("| GPU selection was not in range")
+            gpu_id = click.prompt(
+                f"| Please try again\n| Must be less than {len(gpus)}", type=int
+            )
+        gpu_ind = gpu_id - 1
```

Comment:
I dont fully understand what `gpu_id`a and `gpu_ind` are intended for.
As far as I am aware, the User is meant to input an index (idx) value starting from 0 already. So this here would result in `gpu_ind` values in the range of `{-1 : len(gpus)-1}` - I assume this will break GPU Selection functionality.

From quick testing I did it appears that the Script does not Crash when selecting "0".
But it appears as if the selected device wouldnt perform any tests either.